conftest.py

```diff
@@ -53,3 +55,17 @@ def user() -> dict:
         "zipcode": "12345",
         "city": "example-city"
     }
+
+
+"""Global helper functions."""
+
+
+def pdfToText(pdf) -> str:
+    """Convert pdf bytes into text."""
+    reader = PyPDF2.PdfFileReader(io.BytesIO(pdf))
+
+    result = ""
+    for page in reader.pages:
+        result += page.extract_text() + "\n"
```

Comment:
This is works but is not really pythonic. In Python (list) comprehensions are generally preferred to for loops as they are faster an need less memory. Also strings should mostly not be concatenated by `+` as then in each iteration a new immutable string will be allocated.

  ```suggestion
    result = "\n".join(page.extract_text() for page in reader.pages)
```